### DISCUSSION TITLE
Permissions tab in groups are redundant?

### DISCUSSION THREAD

**Discussion by pgi-jsanchez** [requarks/wiki#6396] [Help / Questions] [answered]
Hello.

I've encountered a configuration issue with the permissions. First I created a group and forgot to add the manage:assets in the permissions tab, but that won't change the page rules permissions after I created the group, then if I create a new page rule, does not copy my "global" permissions from that tab, so, the permissions tab is doing something when I change it there or is redundant and everyone should change that in the page rules? isn't that clear in the current documentation https://docs.requarks.io/en/groups.

Also tried to find if any other discussion, the [3.0 roadmap](https://docs.requarks.io/releases/roadmap) or a proposal in feedback website was something about this, but couldn't find it. Is there any plans to improve the UX or removing the permissions? 

If isn't something in mind, may I suggest a page rule option that syncs with the permissions tab or remove the page permissions from this tab and only have a "system permissions" there.

PD: My permission issue is solve by changing it in the page ruies, but I wonder if things can get improved for better UX by changing them in a certain way.

**Comment by NGPixel** [comment, marked as answer]
The permissions tab acts as a global switch. So you need to enable the permissions you want at the global level, then use Page Rules to apply permissions at a more granular level.

This double permissions system is however indeed confusing for most users and will be removed in v3.
